src/main/java/com/uber/jenkins/phabricator/conduit/Differential.java

```diff
@@ -119,7 +121,13 @@ public String getSummaryMessage(String phabricatorURL) {
 
     public String getPhabricatorLink(String phabricatorURL) {
         String revisionID = this.getRevisionID(true);
-        return String.format("%s%s", phabricatorURL, revisionID);
+        try {
+            URL base = new URL(phabricatorURL);
+            return new URL(base, revisionID).toString();
+        } catch (MalformedURLException e) {
+            e.printStackTrace();
+            return String.format("%s%s", phabricatorURL, revisionID);
```

Comment:
In other words, you're saying that revisionID is something like `"d1"`? I'm struggling to see how that would be the case.